src/bindings/jna/plugins/sorted/src/main/java/org/libelektra/plugin/SortedMetadata.java

```diff
@@ -0,0 +1,35 @@
+package org.libelektra.plugin;
+
+import static java.util.Map.*;
+
+import java.util.Map;
+import org.libelektra.Key;
+import org.libelektra.KeySet;
+import org.libelektra.Plugin;
+
+public class SortedMetadata {
```

Comment:
The class name is confusing. What about simply "Contract"? (That it is for the plugin "Sorted" is imho obvious.)